pygmt/src/tilemap.py

```diff
@@ -111,38 +107,21 @@ def tilemap(
 
     kwargs : dict
         Extra keyword arguments to pass to :meth:`pygmt.Figure.grdimage`.
-
-    Raises
-    ------
-    ImportError
-        If ``rioxarray`` is not installed. Follow
-        :doc:`install instructions for rioxarray <rioxarray:installation>`, (e.g. via
-        ``python -m pip install rioxarray``) before using this function.
     """
     kwargs = self._preprocess(**kwargs)
 
-    if not _HAS_RIOXARRAY:
```

Comment:
The `Figure.tilemap` method no longer requires `rioxarray` explicitly. Instead, `load_tile_map` will raise the error.